### PR TITLE
[bitnami/spark] Adding extra pod labels

### PR DESCRIPTION
**Pull request by wuerzelchen**
**Description of the change**

This PR adds the option to add further labels to master and worker pods. 

**Benefits**

Now you are able to assign specific labels, when you need some. (e.g. in the case of Azure Active Directory Pod Identity you need to assign labels to pods)

**Possible drawbacks**

**Applicable issues**

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/spark/Chart.yaml [hunk 1]
```diff
@@ -21,4 +21,4 @@ name: spark
 sources:
   - https://github.com/bitnami/bitnami-docker-spark
   - https://spark.apache.org/
-version: 4.0.0
+version: 4.0.1
```
Comment (andresbono) [inline bitnami/spark/Chart.yaml:24]: As this is introducing new parameters, can you increase the minor version instead? (second number).

```suggestion
version: 4.1.0
```

File: bitnami/spark/templates/statefulset-worker.yaml [hunk 2]
```diff
@@ -14,6 +14,9 @@ spec:
     metadata:
       labels: {{- include "common.labels.standard" . | nindent 8 }}
         app.kubernetes.io/component: worker
+        {{- if .Values.master.extraPodLabels }}
```
Comment (andresbono) [inline bitnami/spark/templates/statefulset-worker.yaml:17]:
```suggestion
        {{- if .Values.worker.extraPodLabels }}
```

**Review by andresbono — COMMENTED** [review]
Thanks for submitting this PR! Could you please check my suggestions?

File: bitnami/spark/Chart.yaml (hunk 1 above)
Comment (wuerzelchen) [reply bitnami/spark/Chart.yaml:24]: Sure. I just oriented on the pod annotations, there it was a patch, as well.

**Review by andresbono — APPROVED** [review]
LGTM

**Comment by andresbono** [comment]
Thanks for proposing this improvement, @wuerzelchen!! :tada:

**Pull request merged**